web/autograder_service.go

```diff
@@ -117,9 +117,9 @@ func (s *AutograderService) IsAuthorizedTeacher(ctx context.Context, in *pb.Void
 	if err != nil {
```

Comment:
@0xf8f8ff please check this question about admin? I forget the rules for teacher vs admin. It's on the readme, but don't have time to check. But you may have a different intent with this functionality (I think I rewrote it using `getUserAndSCM()` following directly previous semantics.

Reply:
Yes, user must be admin. Updated boolean in method call.

Reply:
About passing course ID: there is no course connection here, scopes are granted by github on the user token issued by Autograder. It happens once per user (unless we revoke all user tokens) and has nothing to do with a specific course.